MODULE.bazel

```diff
@@ -65,22 +73,15 @@ maven = use_extension("@rules_jvm_external//:extensions.bzl", "maven")
 maven.install(
     artifacts = [
         "args4j:args4j:2.33",
-        "com.google.auto.value:auto-value:1.10.4",
-        "com.google.auto.value:auto-value-annotations:1.10.4",
         "com.google.closure-stylesheets:closure-stylesheets:1.5.0",
-        "com.google.code.findbugs:jsr305:3.0.2",
         "com.google.dagger:dagger:2.43.2",
         "com.google.dagger:dagger-compiler:2.43.2",
         "com.google.dagger:dagger-producers:2.43.2",
-        "com.google.guava:guava-testlib:33.2.1-jre",
-        "com.google.guava:guava:33.2.1-jre",
         "com.google.jimfs:jimfs:1.1",
-        "com.google.truth:truth:1.4.0",
         "javax.inject:javax.inject:1",
-        "junit:junit:4.13.2",
         "net.java.dev.javacc:javacc:7.0.13",
-        "org.mockito:mockito-core:5.4.0",
         "org.jsoup:jsoup:1.16.1",
+        "org.mockito:mockito-core:5.4.0",
```

Comment:
google3/third_party/bazel_common/third_party/java/mockito

Though we might need to update bazel_common version.